tests/manage/pv_services/test_ceph_capacity_recovery.py

```diff
@@ -0,0 +1,279 @@
+import logging
+import time
+
+from ocs_ci.ocs.perftests import PASTest
+
+from ocs_ci.ocs import node, constants
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.exceptions import (
+    PodNotCreated,
+    CephHealthException,
+    TimeoutExpiredError,
+)
+
+
+from ocs_ci.ocs.resources import pvc, ocs
+from ocs_ci.ocs.cluster import (
+    get_percent_used_capacity,
+    get_osd_utilization,
+)
+from ocs_ci.utility.utils import TimeoutSampler, ceph_health_check_base
+
+from ocs_ci.ocs.benchmark_operator_fio import get_file_size, BenchmarkOperatorFIO
+
+log = logging.getLogger(__name__)
+
+
+def check_health_status():
+    """
+    Exec `ceph health` cmd on tools pod to determine health of cluster and logs the results
+
+    Returns:
+        boolean: True if HEALTH_OK
+
+    """
+    try:
+        status = ceph_health_check_base()
+        if status:
+            log.info("Health check passed")
+        else:
+            log.info("Health check failed")
+        return status
+    except CephHealthException as e:
+        # skip because ceph is not in good health
+        log.info(f"Ceph health exception received: {e}")
+        return False
+
+
+def get_used_capacity(msg):
```

Comment:
This function can be moved to helpers library.

Reply:
Thanks for your comment. I've moved it to the library (managed_services.py and not to helpers.py to avoid the circular dependency of imports).